src/tests/run_tests.sh

```diff
@@ -23,21 +26,17 @@ QL_IMAGE=quantumleap docker-compose -f ../../docker/docker-compose-dev.yml up -d
 sleep 10
 
 # Backwards Compatibility Test
-docker run -ti --rm --network docker_default \
-    -e ORION_URL="http://orion:1026" \
-    -e QL_URL="http://quantumleap:8668" \
-    quantumleap pytest tests/test_bc.py
+cd ../../
```

Comment:
why is this change of directories needed?  If it's just for the location of the cov-config maybe it's easier to adjust its reference (--cov-config=../../)

Reply:
The work directory location is used for storing the common coverage report ".coverage". It is required to run pytest from work directory and it will append the result on existing coverage report file for each and every submodules.